### ISSUE TITLE
Android version is not working properly?

### ISSUE THREAD

**Issue by KBLNY** [inProgress-team/react-native-orientation-controller#1]
Hi, 
Thanks for your fork!
I encountered an issue with the Android version, the iOS one works perfectly according to your recommendations.
Here is what I am doing : 

```
componentDidMount(){
  Orientation.getDeviceOrientation(
    (orientation, device, size) => {
      console.log(orientation); // undefined
      console.log(device); // undefined
      console.log(size); // undefined

      console.log(arguments); // Sounds weird but get {}
    }
  );
}
```

However, orientation, device and size are all `undefined`.

I checked your Android `ReactOrientationControllerModule`, and I did not find where you send the device and size :

```
params.putString("orientation", Orientation);
thisContext
   .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
   .emit("applicationOrientationDidChange", params);
```

So, firstly, I think you did not handle the device name and size in the Android version, right?

Secondly, if you change the `index.js` to:

```
addApplicationListener: function(callback) {
    return DeviceEventEmitter.addListener(
      'applicationOrientationDidChange', function () {
        callback.apply(this, arguments) // [0] is now removed
      }
    );
},
```

You get the following :

```
componentDidMount(){
  Orientation.getDeviceOrientation(
    (orientation, device, size) => {
      console.log(orientation); // { orientation: 'LANDSCAPE RIGHT' }
      console.log(device); // undefined
      console.log(size); // undefined

      console.log(arguments); // display: { '0': { orientation: 'LANDSCAPE RIGHT' } }
    }
  );
}
```

What do you think about this malfunction? What about the Device and Size?
Device used : Wiko PULP 4G Android 5.1.1 (API 22)

**Comment by julien-inprogress** [comment]
Hi, 
Thanks for the support !
One question : what version of react-native are you using for your project ?

**Comment by jujumoz** [comment]
Hi,
Thanks for the support !
One question : what version of react-native are you using for your project ?

**Comment by julien-inprogress** [comment]
Hi,
Thanks for the support !
One question : what version of react-native are you using for your project ?

**Comment by jusierra** [comment]
Hi,
Thanks for the support !
One question : what version of react-native are you using for your project ?

**Comment by KBLNY** [comment]
I use React Native 0.21.0.
Have you any idea?

**Comment by Mokto** [comment]
Sorry for the delay. We will be looking into this soon.

**Comment by julien-inprogress** [comment]
@KBLNY normally everything is fixed with 0.1.2. Check the README because the usage changed a bit.

**Comment by jusierra** [comment]
@KBLNY : normally everything is fixed with 0.1.2. Check the README because the usage changed a bit.
